<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>788</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Article Browser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202124;
font: 14pt &quot;Roboto&quot;;
color: #BDC1C6;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #303134;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: #1E1F20;
}

QPushButton:onclick {
	background-color: #1A1B1C;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
        <item>
         <widget class="QLabel" name="articleTitle">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 30pt &quot;Roboto Black&quot;;
padding-top: 20px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Article Title</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="articleText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>200</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #303134;
color: #E8EAED;
padding: 20px;
border-radius: 20px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;From: kumar@iopb.ernet.in&lt;br /&gt;Date: Tue Jan 7 07:55:11 1992 (7kb)&lt;br /&gt;Title: An Algorithm to Generate Classical Solutions for String Effective Action Authors: S. Kar, S. Khastgir and A. Kumar Comments: 12 pages Journal-ref: Mod.Phys.Lett. A7 (1992) 1545-1552 &lt;br /&gt;\\ It is shown explicitly, that a number of solutions for the background field equations of the string effective action in space-time dimension D can be generated from any known lower dimensional solution, when background fields have only time dependence. An application of the result to the two dimensional charged black hole is presented. The case of background with more general coordinate dependence is also discussed. \\ From: kumar@iopb.ernet.in&lt;br /&gt;Date: Tue Jan 7 07:55:11 1992 (7kb)&lt;br /&gt;Title: An Algorithm to Generate Classical Solutions for String Effective Action Authors: S. Kar, S. Khastgir and A. Kumar Comments: 12 pages Journal-ref: Mod.Phys.Lett. A7 (1992) 1545-1552 &lt;br /&gt;\\ It is shown explicitly, that a number of solutions for the background field equations of the string effective action in space-time dimension D can be generated from any known lower dimensional solution, when background fields have only time dependence. An application of the result to the two dimensional charged black hole is presented. The case of background with more general coordinate dependence is also discussed. \\ From: kumar@iopb.ernet.in&lt;br /&gt;Date: Tue Jan 7 07:55:11 1992 (7kb)&lt;br /&gt;Title: An Algorithm to Generate Classical Solutions for String Effective Action Authors: S. Kar, S. Khastgir and A. Kumar Comments: 12 pages Journal-ref: Mod.Phys.Lett. A7 (1992) 1545-1552 &lt;br /&gt;\\ It is shown explicitly, that a number of solutions for the background field equations of the string effective action in space-time dimension D can be generated from any known lower dimensional solution, when background fields have only time dependence. An application of the result to the two dimensional charged black hole is presented. The case of background with more general coordinate dependence is also discussed. \\ From: kumar@iopb.ernet.in&lt;br /&gt;Date: Tue Jan 7 07:55:11 1992 (7kb)&lt;br /&gt;Title: An Algorithm to Generate Classical Solutions for String Effective Action Authors: S. Kar, S. Khastgir and A. Kumar Comments: 12 pages Journal-ref: Mod.Phys.Lett. A7 (1992) 1545-1552 &lt;br /&gt;\\ It is shown explicitly, that a number of solutions for the background field equations of the string effective action in space-time dimension D can be generated from any known lower dimensional solution, when background fields have only time dependence. An application of the result to the two dimensional charged black hole is presented. The case of background with more general coordinate dependence is also discussed. \\ From: kumar@iopb.ernet.in&lt;br /&gt;Date: Tue Jan 7 07:55:11 1992 (7kb)&lt;br /&gt;Title: An Algorithm to Generate Classical Solutions for String Effective Action Authors: S. Kar, S. Khastgir and A. Kumar Comments: 12 pages Journal-ref: Mod.Phys.Lett. A7 (1992) 1545-1552 &lt;br /&gt;\\ It is shown explicitly, that a number of solutions for the background field equations of the string effective action in space-time dimension D can be generated from any known lower dimensional solution, when background fields have only time dependence. An application of the result to the two dimensional charged black hole is presented. The case of background with more general coordinate dependence is also discussed. \\&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fullArticleLinkLabel">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 87 15pt &quot;Roboto Bold&quot;;
padding-left: 10px;</string>
          </property>
          <property name="text">
           <string>Open Full Article</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="randomArticleButton">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Random</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="likeButton">
            <property name="text">
             <string>Like</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0" colspan="2">
         <widget class="QListView" name="recomView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="logInButton">
          <property name="text">
           <string>Log In</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="signInButton">
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="recommendationsLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>You may also like:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
